config/deployments/integration-tests/README.md

```diff
@@ -0,0 +1,15 @@
+# Integration tests
```

Comment:
Should we add a link to this file on the dependency in the go mod file?

(we might have to move vm-operator to a separate require block for that, but why not)